### ISSUE TITLE
Missing example program

### ISSUE THREAD

**Issue by Heath123** [debrouxl/tilibs#71]
https://github.com/debrouxl/tilibs/blob/master/libticables/trunk/docs/html/htlib.html says:

> You will find in the <i>test</i> folder of the library source archive a test/example program which uses this lib.

That file doesn't exist though

**Comment by adriweb** [comment]
you're probably looking for the source files located in the `tests` folder of each lib. For instance https://github.com/debrouxl/tilibs/blob/master/libticalcs/trunk/tests/test_ticalcs_2.cc

**Comment by Heath123** [comment]
I was looking for the full version of the example program in the HTML file, I'm not sure if I need it any more though

**Comment by debrouxl** [comment]
Indeed, it's best to use the test_ti*.c* files from the `tests` folders, especially `test_ticalcs_2`, which is by far the most used and best tested one. The version on the experimental2 branch is significantly more powerful, and therefore an even better example.
